src/Administration/Resources/administration/src/module/sw-customer/page/sw-customer-detail/index.js

```diff
@@ -113,13 +113,17 @@ Component.register('sw-customer-detail', {
     },
 
     methods: {
+        defaultCriteria() {
```

Comment:
Please add it as a computed property instead of a method,
take a look here for the reasons: https://vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods